db/migrate/20160614093540_unpublish_stale_placeholders.rb

```diff
@@ -0,0 +1,18 @@
+class UnpublishStalePlaceholders < ActiveRecord::Migration
+  def change
+    stale_placeholders_hash = {
+      "12a4eb7a-6037-4cc0-aa58-0a4f2fbc5e7f" => "5d3b5d66-7631-11e4-a3cb-005056011aef",
+      "e0deb0ec-e9fc-4308-b8c0-eba4dc92aa83" => "5d37acc4-7631-11e4-a3cb-005056011aef",
+      "f656d065-43aa-4ab0-91f7-a6809ce5b68b" => "5d645268-7631-11e4-a3cb-005056011aef",
+    }
+
+    stale_placeholders_hash.each do |old, new|
+      Services.publishing_api.unpublish(
+        old,
+        type: "redirect",
+        alternative_path: new,
```

Comment:
Good catch.
wtb type system :(

Just to be clear, `new` is not a keyword, just a method, right?